### PR TITLE
techops10133: fix multiple search bars on a page

### PR DESCRIPTION
**Pull request by jesuyedavid**

### CODE REVIEW

**Review by benmcginnis — APPROVED** [review]
lgtm, are there any other ids that we need to scope like this?  also have you tried running your test page through any accessibility tools like Chrome Lighthouse?

File: src/ui/templates/search/search.hbs [hunk 1]
```diff
@@ -23,10 +23,10 @@
           aria-label="{{labelText}}"
           aria-autocomplete="list"
           aria-controls="{{autocompleteContainerIdName}}"
-          aria-describedby="instructions"
+          aria-describedby="{{searchBarContainerId}}-instructions"
           aria-haspopup="listbox"
         >
-        <div id="instructions" class="yxt-SearchBar-instructions sr-only">
+        <div id="{{searchBarContainerId}}-instructions" class="yxt-SearchBar-instructions sr-only">
           {{translate
```
Comment (cea2aj) [inline src/ui/templates/search/search.hbs:30]: does this solution use the autocompleteContainerIdName at all?

Comment (jesuyedavid) [reply src/ui/templates/search/search.hbs:30]: yes, i see it on line 25. if that's what you are asking?

Comment (cea2aj) [reply src/ui/templates/search/search.hbs:30]: ah that makes sense

**Pull request merged**